automation/default_browser_params.json

```diff
@@ -7,6 +7,8 @@
     "cookie_instrument": false,
     "js_instrument": false,
     "cp_instrument": false,
+    "http_instrument": false,
```

Comment:
I'm curious why `proxy` is true by default and `http_instrument` is false. Do you plan to make `http_instrument` default later?